exla/livebooks/rotating_image.livemd

```diff
@@ -0,0 +1,224 @@
+# Rotating Image
+
+## Imports
```

Comment:
We usually call the first section "Setup" and keep all dependencies, imports, configs there. Just a convention :)

```suggestion
## Setup
```